sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -433,7 +484,8 @@ def _collect_plans(self, rows, cursor, deadline):
                             "collection_errors": collection_errors,
                         },
                         "query_signature": row['query_signature'],
```

Comment:
What is this query signature referring to?

Reply:
the `query_signature` of the `statement_text`

Reply:
If it's referring to the query signature of only one of the queries in the procedure then I don't think we should have it on the event at all. Could lead to confusion if someone searches for plans matching that query signature and sees this procedure plan, then wonders why that lookup only works sometimes.